### PR TITLE
added figaro as a secrets manager

### PR DESCRIPTION
**Pull request by nalnaji**
Hey guys,

Figaro is a gem that makes installing and setting up or API Secrets really easy. This will also help us when we are deploying Heroku, since we can give it our secrets through figaro as well :)

Reviewer: @DarryQueen, @nishadsingh1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,6 @@
 
 # Ignore cloud9 metadata.
 .c9/*
+
+# Ignore application configuration
+/config/application.yml
```
Comment (DarryQueen) [inline .gitignore]: This isn't it. I believe it's `keys.rb`.

Comment (DarryQueen) [inline .gitignore]: Just kidding, I realized that this was add. You should remove `keys.rb` though.

File: Gemfile [hunk 2]
```diff
@@ -22,6 +22,8 @@ gem 'devise'
 gem 'omniauth-facebook'
 gem 'omniauth-google-oauth2'
 
+# This gem handles API Secrets :)
```
Comment (DarryQueen) [inline Gemfile]: Stupid emoticon.

**Comment by DarryQueen** [comment]
Refer to comments. You might want to quickly document how we add API keys through Figaro.

File: .gitignore (hunk 1 above)
Comment (nalnaji) [inline .gitignore]: This is necessary. I will delete keys.rb though.

**Comment by DarryQueen** [comment]
LGTM.

**Pull request merged**